src/adagrams.js

```diff
@@ -1,15 +1,84 @@
+import {
+  LETTER_POOL, 
+  SCORE_MAP
+} from "../src/data.js";
+
 export const drawLetters = () => {
   // Implement this method for wave 1
+  const drawPool = [];
+  for (const [key, value] of Object.entries(LETTER_POOL)) {
+    for (let i = 0; i < value; i++) {
+      drawPool.push(key);
+    }
+  }
+  const hand = []; 
+  for (let i = 0; i < 10; i++) {
+    const pieceIndex = Math.floor(Math.random() * drawPool.length);
+    const piece = drawPool[pieceIndex];
+    hand.push(piece);
+    const index = drawPool.indexOf(piece);
```

Comment:
Small note: The value of `index` here is not the same as `pieceIndex` because the method indexOf returns the first index of `piece` that it finds. 

For example if pieceIndex = 26 and the value is `E` the indexOf would return 17 for `index` so you're not actually deleting the 'same piece'. Could you use pieceIndex itself to splilce from drawPool?